src/pages/terms/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import { Page, Container, Padding } from '../../components/layout'
+import { Button, Primary } from '../../components/button'
+import { HicetnuncContext } from '../../context/HicetnuncContext'
+import { BottomBanner } from '../../components/bottom-banner'
+import { getLanguage } from '../../constants'
+import styles from './styles.module.scss'
+
+import Markdown from 'markdown-to-jsx';
+
+export class Terms extends Component {
+  static contextType = HicetnuncContext
+
+  language = getLanguage()
+
+  docLang = 'en'
+
+  // load public\languages\documents\terms-en.md based on language
+  componentDidMount() {
+    this.loadDocument()
+  }
+
+  // load public\languages\documents\terms-en.md based on language
+  componentDidUpdate(prevProps) {
+    if (this.props.docLang !== prevProps.docLang) {
+      this.loadDocument()
+    }
+  }
+
+  loadDocument() {
+    const docLang = this.docLang
+    console.log(this.docLang)
+    const document = `/languages/documents/terms-${docLang}.md`
+    fetch(document)
+      .then(response => response.text())
+      .then(text => {
+        console.log(text)
+        this.termsContent = text
+        this.forceUpdate()
```

Comment:
termsContent could be stored in the react state, then forceUpdate() would not be needed.